styles/dialog.less

```diff
@@ -75,25 +64,12 @@
 		color: @text-color-subtle;
 	}
 
-	select {
-		-webkit-appearance: none;
-		appearance: none;
-	}
-
-	button {
-		background: transparent;
-		border: 1px solid transparent;
-		border-radius: 4px;
-		color: @text-color;
-		margin-right: 10px;
-
-		&:focus,
-		&:not([disabled]):hover {
-			color: @text-color-highlight;
-		}
-
-		&[disabled] {
-			color: @text-color-subtle
+	.create-branch {
+		label.input-label,
+		input[type=text],
+		select {
+			display: block;
+			width: 100%;
```

Comment:
these styles should probably be on every dialog not only .create-branch

and if you do need to create styles specific to a certain dialog create a new file called [DialogName].less similar to the CommitDialog.less